docs/quick_tutorial/authentication.rst

```diff
@@ -50,7 +50,9 @@ Steps
     :linenos:
 
 #. Create an ``authentication/tutorial/security.py`` module that can find our
-   user information by providing an *authentication policy callback*:
+   user information by providing an *authentication policy callback*.
+   One-way hash function ``hash_password_with_salt`` is used to avoid user's
+   password stored in plain text:
```

Comment:
It is preferable to explain what something does, instead of what it avoids. Suggest:

"The function ``hash_password_with_salt`` hashes the concatenation of the user's password and a randomly generated salt."